extensions/skins/conversation_v1/static/conversation.css

```diff
@@ -62,6 +62,11 @@ html, body {
 .conversation-skin-oppia-avatar {
   left: -20px;
 }
+.conversation-skin-tutor-card-top-section .conversation-skin-oppia-avatar {
+  height: 48px;
+  left: -28px;
```

Comment:
Also, I have to say, the large 'O' icon doesn't look very nice. I've grown quite accustomed to the otter.

What do you say we just put it in there?